pmd-java/src/main/resources/category/java/bestpractices.xml

```diff
@@ -802,6 +802,76 @@ public class MyTest {
         </example>
     </rule>
 
+    <rule name="JUnit5TestShouldBePackagePrivate"
+          language="java"
+          since="6.35"
+          message="JUnit 5 tests that use @Test, @RepeatedTest, @TestFactory, @TestTemplate or @ParameterizedTest should be package private."
```

Comment:
```suggestion
          message="JUnit 5 tests should be package-private."
```

I don't know of Junit 5 tests that don't use one of these annotations.

Reply:
Done